scripts/systemd/falco-ebpf.service

```diff
@@ -1,13 +1,14 @@
 [Unit]
-Description=Falco: Container Native Runtime Security
+Description=Falco: Container Native Runtime Security with ebpf
 Documentation=https://falco.org/docs/
+PartOf=falco@ebpf.target
+Conflicts=falco-kmod.service
```

Comment:
If you enable `falco@ebpf.target` it will automatically disable `falco@kmod.target`.